frb_codegen/src/ir/ty_delegate.rs

```diff
@@ -14,14 +14,23 @@ pub enum IrTypeTime {
 
 #[cfg(feature = "chrono")]
 impl IrTypeTime {
-    fn safe_ident(&self) -> &str {
+    #[inline]
+    pub fn safe_ident(&self) -> &str {
         match self {
             IrTypeTime::Local => "Local",
             IrTypeTime::Utc => "Utc",
             IrTypeTime::Duration => "Duration",
             IrTypeTime::Naive => "Naive",
         }
     }
+    #[inline]
+    pub fn is_duration(&self) -> bool {
+        matches!(self, Self::Duration)
+    }
+    #[inline]
+    pub fn is_utc(&self) -> bool {
+        matches!(self, Self::Naive | Self::Utc)
```

Comment:
uber-nit: are we sure that "naive" is "is_utc"?

Reply:
This should be from the original implementation IIRC, so I left it as is.